ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md

```diff
@@ -246,13 +257,22 @@ interface Vlan42
    ip helper-address 10.10.64.150 source-interface Loopback0
    ip helper-address 10.10.96.150 source-interface Loopback0
    ip helper-address 10.10.96.151 source-interface Loopback0
+   isis enable EVPN_UNDERLAY
+   isis authentication mode sha key-id 5 level-1
+   isis authentication mode sha key-id 5 level-2
    ip address virtual 10.10.42.1/24
 !
 interface Vlan43
    description SVI Description
    no shutdown
    ipv6 dhcp relay destination a0::2 vrf TEST local-interface Loopback44 link-address a0::4
    ipv6 address a0::1/64
+   isis authentication key-id 2 algorithm sha-512 key 0 password
```

Comment:
the config should short with respect to key-id, can you please check if that can be done?

Reply:
This is difficult to implement  as it requires to combine all key from both, level_1 and level_2 in a list and then sort. Currently the code sorts within the levels both, level_1 and level_2.